src/main/java/org/opensearch/timeseries/TimeSeriesAnalyticsPlugin.java

```diff
@@ -1672,6 +1679,17 @@ public List<NamedXContentRegistry.Entry> getNamedXContent() {
             );
     }
 
+    @Override
+    public Collection<SystemIndexDescriptor> getSystemIndexDescriptors(Settings settings) {
+        List<SystemIndexDescriptor> systemIndexDescriptors = new ArrayList<>();
+        systemIndexDescriptors.add(new SystemIndexDescriptor(ALL_AD_RESULTS_INDEX_PATTERN, "Time Series Analytics Results index pattern"));
```

Comment:
We have forecasting now. Can you change to "AD result index pattern"?

Reply:
Changed